spec/lib/metasploit/framework/login_scanner/jenkins_spec.rb

```diff
@@ -2,9 +2,103 @@
 require 'metasploit/framework/login_scanner/jenkins'
 
 RSpec.describe Metasploit::Framework::LoginScanner::Jenkins do
+  subject(:http_scanner) { described_class.new('host' => 'nosuchplace.metasploit.com') }
 
-    it_behaves_like 'Metasploit::Framework::LoginScanner::Base',  has_realm_key: true, has_default_realm: false
-    it_behaves_like 'Metasploit::Framework::LoginScanner::RexSocket'
-    it_behaves_like 'Metasploit::Framework::LoginScanner::HTTP'
```

Comment:
is it possible to keep these tests around? I'm guessing that the behavior should be the same for the scanners I'd imagine 👀